CMake/Modules/FindPhysicsFS.cmake

```diff
@@ -0,0 +1,59 @@
+#.rst:
```

Comment:
Why duplicate the FindPhysicsFS.cmake module from CMake? It looks like you're trying to do some things specific to your setup; I'd instead recommend using facilities built in to the vanilla CMake module. Take a look at https://github.com/Kitware/CMake/blob/master/Modules/FindPhysFS.cmake. Looks like you should be able to set `PHYSFSDIR` environment variable.